apps/condo/domains/common/utils/date.js

```diff
@@ -1,8 +1,15 @@
 const dayjs = require('dayjs')
+const customParseFormat = require('dayjs/plugin/customParseFormat')
+const { isEmpty, isString } = require('lodash')
+
+dayjs.extend(customParseFormat)
 
 const DATE_FORMAT = 'YYYY-MM-DD'
 const DATE_FORMAT_Z = 'YYYY-MM-DDTHH:mm:ss.SSSZZ'
 
+const SYMBOLS_TO_CUT_FROM_DATES_REGEXP = /[^-TZ_:.,( )/\d]/gi // -_:.,()/ recognizable by dayjs https://day.js.org/docs/en/parse/string-format
+const UTC_END_REGEXP = /(Z|(\.\d{3}[+-](\d{2}:?\d{2})))$/ // Z | +5000 | +50:00
```

Comment:
Whats this used for? We're trying to avoid manual string parsing. I get the "remove trash" part from above, but what's the use of this one?

Reply:
We have to do these hacks since dayjs couldn't do everything himself.
Moved this from common to specific utils thou